masterlist.yaml

```diff
@@ -1805,6 +1805,21 @@ plugins:
         condition: 'checksum("KFE - Refill Water Bottles.esp", B85EF60C)'
 
 ###### Gameplay - Combat ######
+  - name: 'BLEED.esp'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/61141/'
+        name: 'BLEED TTW'
+    req:
+      - *JIPLN
+    msg: [ *requires_xNVSE ]
+    tag:
+      - Enchantments
+      - EnchantmentStats
```

Comment:
![grafik](https://github.com/loot/falloutnv/assets/10406657/0f35931d-22b9-491d-9f4f-97846c0ff485)

Was this record the only reason `EnchantmentStats` has been added?